pages/am/3.x/user-guide/branding/user-guide-branding-internationalization.adoc

```diff
@@ -0,0 +1,60 @@
+= Internationalization
+:page-sidebar: am_3_x_sidebar
+:page-permalink: am/current/am_userguide_branding_internationalization.html
+:page-folder: am/user-guide
+:page-layout: am
+
+[label label-version]#New in version 3.19#
+
+== Overview
+
+AM is aiming to support multiple languages for the end user convenience.
```

Comment:
AM supports internationalization in multiple languages so that end users can benefit from a great user experience.